src/test/java/edu/ucsb/courses/controllers/BasicSearchControllerTests.java

```diff
@@ -50,4 +67,27 @@ public void test_basicSearch() throws Exception {
         assertEquals(expectedResult, responseString);
     }
 
+    // @Test
+    // public void test_csvDownload() throws Exception {
+    //     ArrayList<Course> courses = new ArrayList<Course>();
+    //     Course course = new Course();
+    //     course.setQuarter("20104");
+    //     course.setCourseId("CMPSC 156");
+    //     course.setTitle("ADV APP PROG");
+    //     course.setDescription("best course ever");
+    //     courses.add(course);
+    //     CoursePage cp = new CoursePage(0, 0, 0, courses);
+    //     String cpJSON = mapper.writeValueAsString(cp);
+    //     logger.info("cpJSON = {}", cpJSON);
+    //     String urlTemplate = "/api/public/basicsearch/downloadcsv?qtr=%s&dept=%s&level=%s";
+    //     String url = String.format(urlTemplate, "20204", "CMPSC", "L");
+    //     when(ucsbCurriculumService.getJSON(any(String.class), any(String.class), any(String.class)))
+    //             .thenReturn(cpJSON);
+    //     // when(downloadCsvService.listToCSV(any(String.class), any(PrintWriter.class)));
+    //     MvcResult response = mockMvc.perform(get(url).contentType("text/csv")).andExpect(status().isOk())
+    //             .andReturn();
+    //     String responseString = response.getResponse().getContentAsString();
+    //     assertEquals("", responseString);
+    // }
+
```

Comment:
Is it necessary to keep this commented code in the final

Reply:
^^^ What he said